js/ftx.js

```diff
@@ -2485,11 +2485,27 @@ module.exports = class ftx extends Exchange {
     async fetchBorrowRateHistories (since = undefined, limit = undefined, params = {}) {
         await this.loadMarkets ();
         const request = {};
-        const endTime = this.safeNumber2 (params, 'till', 'end_time');
+        let endTime = this.safeNumber2 (params, 'till', 'end_time');
+        if (limit > 48) {
+            throw new BadRequest (this.id + ' fetchBorrowRateHistories limit cannot exceed 48');
+        }
+        if ((endTime - since) > 172800000) {
+            throw new BadRequest ('The time between since and the end time be >= 48 hours using ' + this.id + ' fetchBorrowRateHistories');
+        }
         if (since !== undefined) {
             request['start_time'] = since / 1000;
             if (endTime === undefined) {
-                request['end_time'] = this.milliseconds () / 1000;
+                const now = this.milliseconds ();
+                const sinceLimit = (limit === undefined) ? 2 : limit;
+                endTime = Math.min ((since + (3600000 * (sinceLimit - 1))), now);
```

Comment:
Let's split this expression, cause it violated this guideline:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

Also, if possible, please, avoid magic numbers, better add named constants or comments.